lib/js/src/session/SdlSession.js

```diff
@@ -300,7 +301,7 @@ class SdlSession {
      * @return {VideoStreamingParameters} the desired video streaming parameters
      */
     getDesiredVideoParams () {
-        if (!this._desiredVideoParams) {
+        if (this._desiredVideoParams === null || this._desiredVideoParams === undefined) {
```

Comment:
Why not perform an `instanceof` check?

Reply:
I will try doing that instead. It will be better to check for VideoStreamingParameters.